dev/reference/tools/br/use-cases.md

```diff
@@ -1,80 +1,75 @@
 ---
-title: BR 快速备份恢复最佳实践
-category: how-to
+title: BR 备份与恢复场景示例
+category: reference
+aliases: ['/docs-cn/dev/how-to/maintain/backup-and-restore/br-best-practices/']
```

Comment:
@TomShawn I find it lacks one alias for `/dev/reference/tools/br/br-best-practices.md`. This lead to a 404 for the original link: https://pingcap.com/docs-cn/dev/reference/tools/br/br-best-practices/

Please help update it. Thanks!